controllers/profile_controller.js

```diff
@@ -24,15 +25,15 @@ const controller = {
   //Method POST: to Post data from Personal information Form
   async create(req, res) {
     const personalData = req.body;
-    const currentUser = await Users.findById(`${req.session?.currentUser._id}`);
-    await currentUser.updateOne({ personalData });
-    res.redirect(`/profiles/${currentUser._id}`);
+    const profileOwner = await Users.findById(`${req.session?.profileOwner._id}`);
```

Comment:
Why is the data from `session`?

Reply:
<img width="735" alt="image" src="https://user-images.githubusercontent.com/86551216/183241834-830d0199-9f0b-4011-8347-a0341521bb59.png">
because in that page, in the URL, don't have the user_id. 
URL to create new personal data POST:
http://localhost:8080/profiles/new

Reply:
then, this means your url is wrong.
this function / page seems like creating a new user. creating a new user means sign up.

*confusing*

Reply:
okay, I will add the ID of user when create personal data.